### ISSUE TITLE
Code convention issue

### ISSUE THREAD

**Issue by maucon** [TheSwerik/CalcPrimesMultiThread#4]
CalcPrimesMultiThread/Backend/src/Prime/Task/StaticPrime.cs 
Starting at line 37
```
for (var i = (root & 1) == 0 ? root + 1 : root + 2;
     i <= n && !(token?.IsCancellationRequested ?? false);
     i += 2)
   if (!sieve[i]) primes.Add(i);
return primes;
```

Please add an empty line before return statement for improved readability.

**Comment by TheSwerik** [comment]
oof

**Comment by TheSwerik** [comment]
fixed in 6a19244
